spec/services/user_alerts/alert_user_about_account_verified_spec.rb

```diff
@@ -38,7 +38,9 @@
         expect_delivered_email(
           to: [user.confirmed_email_addresses.first.email],
           subject: t('user_mailer.account_verified.subject', app_name: APP_NAME),
-          body: ['<table class="button expanded large radius">', 'localhost:3000'],
+          body: [
+            'http://www.example.com/redirect/return_to_sp/account_verified_cta',
```

Comment:
Turns out `body:` is only comparing against the plaintext version of the email, so checking for HTML here fails.

Previously, we weren't correctly processing these body checks, so this was erroneously passing prior to this PR

Reply:
I sometimes wonder how many tests erroneously pass and conceal bugs. 😬 Glad you found this.